### DISCUSSION TITLE
Unable to Login - Need help Asap

### DISCUSSION THREAD

**Discussion by stephenasuncionDEV** [vercel/community#1755] [Help] [answered]
### Summary

Hey,

I registered with an email alias which forwards to an outlook email. I recently migrated to a different domain and forgot to update my Vercel email. I am now logged off after I cleared my cache and cookies and I can't log back in because I removed the email forwarding.

How can I get my account back? I sent an email to registration@vercel.com already but it's almost 3 days now.

Thank you so much!
Stephen

**Comment by amyegan** [comment, marked as answer]
Hi @stephenasuncionDEV. It looks like there was a response to your case today so you should have an email waiting in your inbox 🙏
